[2025-10-11T04:44:26.481+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-11T04:44:26.534+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline_flow.load_users manual__2025-10-09T12:26:00.230217+00:00 [queued]>
[2025-10-11T04:44:26.548+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline_flow.load_users manual__2025-10-09T12:26:00.230217+00:00 [queued]>
[2025-10-11T04:44:26.550+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 3
[2025-10-11T04:44:26.576+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_users> on 2025-10-09 12:26:00.230217+00:00
[2025-10-11T04:44:26.590+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=7250) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-11T04:44:26.592+0000] {standard_task_runner.py:64} INFO - Started process 7261 to run task
[2025-10-11T04:44:26.590+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipeline_flow', 'load_users', 'manual__2025-10-09T12:26:00.230217+00:00', '--job-id', '43', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/tmp/tmpf8mikk9_']
[2025-10-11T04:44:26.593+0000] {standard_task_runner.py:91} INFO - Job 43: Subtask load_users
[2025-10-11T04:44:26.671+0000] {task_command.py:426} INFO - Running <TaskInstance: pipeline_flow.load_users manual__2025-10-09T12:26:00.230217+00:00 [running]> on host 321597a50810
[2025-10-11T04:44:26.781+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='oluwaseyiogundimu10@gmail.com' AIRFLOW_CTX_DAG_OWNER='seyi' AIRFLOW_CTX_DAG_ID='pipeline_flow' AIRFLOW_CTX_TASK_ID='load_users' AIRFLOW_CTX_EXECUTION_DATE='2025-10-09T12:26:00.230217+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-09T12:26:00.230217+00:00'
[2025-10-11T04:44:26.783+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-11T04:44:26.797+0000] {read_write_buckets.py:22} INFO - Reading users.csv from movie-pipeline-bronze bucket.
[2025-10-11T04:44:26.936+0000] {s3_client.py:30} INFO - âœ… Successfully initialized S3 client.
[2025-10-11T04:44:27.778+0000] {read_write_buckets.py:34} INFO - Successfully read users.csv into DataFrame. Shape: (943, 5)
[2025-10-11T04:44:27.779+0000] {transform.py:181} INFO - Starting transformations for users..
[2025-10-11T04:44:27.821+0000] {transform.py:198} INFO - Users data cleaned with 943 records
[2025-10-11T04:44:27.823+0000] {read_write_buckets.py:50} INFO - Writing users.csv to movie-pipeline-silver bucket.
[2025-10-11T04:44:29.143+0000] {read_write_buckets.py:66} INFO - Successfully wrote users.csv to Silver bucket.
[2025-10-11T04:44:29.145+0000] {watermarks.py:48} INFO - Updating watermarks in S3...
[2025-10-11T04:44:29.147+0000] {watermarks.py:25} INFO - Reading watermarks from S3 Bronze bucket...
[2025-10-11T04:44:29.383+0000] {watermarks.py:35} WARNING - Watermarks not found in S3. Initializing new one. Details: An error occurred (NoSuchKey) when calling the GetObject operation: The specified key does not exist.
[2025-10-11T04:44:29.397+0000] {logging_mixin.py:188} WARNING - /opt/***/dags/../pipeline/b_silver/watermarks.py:63 FutureWarning: The behavior of DataFrame concatenation with empty or all-NA entries is deprecated. In a future version, this will no longer exclude empty or all-NA columns when determining the result dtypes. To retain the old behavior, exclude the relevant entries before the concat operation.
[2025-10-11T04:44:29.831+0000] {watermarks.py:76} INFO - Watermarks updated successfully in S3.
[2025-10-11T04:44:29.833+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-11T04:44:29.834+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-11T04:44:29.855+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipeline_flow, task_id=load_users, run_id=manual__2025-10-09T12:26:00.230217+00:00, execution_date=20251009T122600, start_date=20251011T044426, end_date=20251011T044429
[2025-10-11T04:44:29.903+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-10-11T04:44:29.942+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-11T04:44:29.946+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
